tests/resolvers/Mutation/createEvent.spec.ts

```diff
@@ -166,18 +185,18 @@ describe("resolvers -> Mutation -> createEvent", () => {
         organizationId: testOrganization?.id,
         allDay: false,
         description: "newDescription",
-        endDate: new Date().toUTCString(),
+        endDate: new Date("2023-01-29T00:00:00Z"),
```

Comment:
Will this test work after `2023-01-29T00:00:00Z`?

Reply:
Yes, it will work for any date. This is the  sample date that  was used to create multiple events for verification, but the test is not restricted to that specific date and will function for any date provided.
I have taken this so that n number of events are generated and same is verified

![image](https://github.com/PalisadoesFoundation/talawa-api/assets/76515568/16bc8ca9-7e6a-449f-9c2b-4d6779a76e01)